config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -115,3 +115,53 @@ periodics:
           # this is mostly for building kubernetes
           memory: 9Gi
           cpu: 7
+- interval: 6h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-compatibility-versions-feature-gate-test
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: compatibility-versions-feature-gate-test
+    description: Uses kind to run bespoke feature gate tests from the n-1 kubernetes release yaml files against a latest kubernetes master components w/ --emulated-version=n-1 set.
+    # TODO(aaron-prindle) route the alert email to a rotation vs individual email
```

Comment:
probably also more owners to experiment/compatiblity-versions if/when we change this

Reply:
Created https://github.com/kubernetes/test-infra/issues/34269 and updated the TODO description to state both tasks